example/driver/simpledriver.go

```diff
@@ -83,6 +83,15 @@ func (s *SimpleDriver) HandleReadCommands(deviceName string, protocols map[strin
 		if reqs[0].DeviceResourceName == "SwitchButton" {
 			cv, _ := dsModels.NewBoolValue(reqs[0].DeviceResourceName, now, s.switchButton)
 			res[0] = cv
+		} else if reqs[0].DeviceResourceName == "Xrotation" {
```

Comment:
Are these already in the device profile?

Reply:
Yes
https://github.com/edgexfoundry/device-sdk-go/blob/1d7933a5c524335d3951382295f086a24d5103f3/example/cmd/device-simple/res/Simple-Driver.yaml#L26-L32
So I modified the corresponding driver code to make sure deviceResource can be accessed directly.